src/pybind/mgr/cephadm/templates/services/haproxy/haproxy.cfg.j2

```diff
@@ -0,0 +1,45 @@
+# {{ cephadm_managed }}
+global
+    log         127.0.0.1 local2
+    chroot      /var/lib/haproxy
+    pidfile     /var/run/haproxy.pid
+    maxconn     8000
+    daemon
+    stats socket /var/lib/haproxy/stats
+
+defaults
+    mode                    http
+    log                     global
+    option                  httplog
+    option                  dontlognull
+    option http-server-close
+    option forwardfor       except 127.0.0.0/8
+    option                  redispatch
+    retries                 3
+    timeout http-request    1s
+    timeout queue           20s
+    timeout connect         5s
+    timeout client          1s
+    timeout server          1s
+    timeout http-keep-alive 5s
+    timeout check           5s
+    maxconn                 8000
+
+frontend stats
+    bind *:{{ spec.ha_proxy_port }}
+    stats {{ ha_proxy_stats_enabled }}
+    stats uri /stats
+    stats refresh 10s
+    stats auth {{ spec.ha_proxy_stats_user }}:{{ spec.ha_proxy_stats_password }}
+    http-request use-service prometheus-exporter if { path /metrics }
```

Comment:
i think ha_proxy_enable_prometheus_exporter has something to do with this line but i need more details